server.py

```diff
@@ -137,6 +137,7 @@ def archives():
 if __name__ == "__main__":
     app.debug = True
 # Change the postgresql info below username, password, port
-    connect_to_db(app, "postgresql://username:password@localhost/newb")
-    app.run(port=5000)
+    connect_to_db(app, "postgresql:///newb")
```

Comment:
are these changes to the server intended? If so, what do they do? Same question for line 23 in the model :)